contracts/test/DepositStub.sol

```diff
@@ -77,4 +78,8 @@ contract DepositStub is IDeposit {
     function withdrawableAmount() external view override returns (uint256) {
         return address(this).balance;
     }
+
+    function setAuctionValue(uint256 _auctionValue) external {
+        auctionValue = _auctionValue;
```

Comment:
Linter is complaining about the function order.

Reply:
If we want to address https://github.com/keep-network/coverage-pools/pull/79#discussion_r644598509 it must be at the end. After all, it's just a warning 🤷

Reply:
... and it's a warning in a stub. Yeah :+1: